lib/src/comment_references/parser.dart

```diff
@@ -5,6 +5,67 @@
 import 'package:charcode/charcode.dart';
 import 'package:meta/meta.dart';
 
+const _operatorKeyword = 'operator';
+const Map<String, String> operatorNames = {
+  '[]': 'get',
+  '[]=': 'put',
+  '~': 'bitwise_negate',
+  '==': 'equals',
+  '-': 'minus',
+  '+': 'plus',
+  '*': 'multiply',
+  '/': 'divide',
+  '<': 'less',
+  '>': 'greater',
+  '>=': 'greater_equal',
+  '<=': 'less_equal',
+  '<<': 'shift_left',
+  '>>': 'shift_right',
+  '>>>': 'triple_shift',
+  '^': 'bitwise_exclusive_or',
+  'unary-': 'unary_minus',
+  '|': 'bitwise_or',
+  '&': 'bitwise_and',
+  '~/': 'truncate_divide',
+  '%': 'modulo'
+};
+
+class StringTrie {
+  final Map<int, StringTrie> children = {};
+  bool valid = false;
```

Comment:
Would you mind documenting this field? I think I know what it means, but maybe not in the context of a field on a whole trie.

Reply:
done